osu.Game/Screens/Play/PlayerSettings/InputSettings.cs

```diff
@@ -19,12 +19,12 @@ public InputSettings()
             {
                 mouseButtonsCheckbox = new PlayerCheckbox
                 {
-                    LabelText = "Disable mouse buttons"
+                    LabelText = "Mouse buttons"
```

Comment:
"Mouse buttons" -> "Enable mouse buttons"

Reply:
It seems unnecessary. For consistency if this gets approved, the other toggles should have it too(i.e. `Enable storyboards`, etc.)

Reply:
Oh yeah you're right, it a makes more sense in the context of the settings box than in code.